llvm/lib/SYCLLowerIR/SYCLJointMatrixTransform.cpp

```diff
@@ -11,16 +11,20 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "llvm/SYCLLowerIR/SYCLJointMatrixTransform.h"
+#include <queue>
 
 #include "llvm/IR/IRBuilder.h"
+#include "llvm/SYCLLowerIR/SYCLJointMatrixTransform.h"
```

Comment:
Main module header should go first, see [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#include-style)